packages/core/strapi/lib/commands/controllers/list.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const CLITable = require('cli-table3');
+const chalk = require('chalk');
+
+const strapi = require('../../index');
+
+module.exports = async function() {
+  const app = await strapi().register();
```

Comment:
Just to be sure, we don't need to `load()` here? Like the `routes:list` command?

Reply:
yes just the register phase is enough. the routes are only registered in the router during load that's why